src/assets/icons/eye.svg

```diff
@@ -0,0 +1,3 @@
+<svg width="20" height="14" viewBox="0 0 20 14" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
If you need to add a new icon, please make sure its consistent with the other icons.
Icons are stored without `width` and `height`, so we can adjust it with css, as well as `fill` set to `black`. On all other child nodes of the svg we should remove the `fill` attribute, so we can use again tailwind to style the color